imports/ui/pages/panel/jobs/show.js

```diff
@@ -2,7 +2,18 @@ import {Template} from 'meteor/templating';
 
 Template.showJob.events({
   'click #apply': function() {
-    Meteor.call('applyToJob', FlowRouter.getParam('id'));
+    var btn = $(this)
+    btn.button('loading');
+
+    Meteor.call('applyToJob', FlowRouter.getParam('id'), function(error, result) {
+      if (error) {
+        Meteor.call('displayErrors', error);
```

Comment:
Or even better, if what I said is true, it would be better to call it outside the if statement